analysis_schema/data_classes.py

```diff
@@ -3,9 +3,10 @@
 
 from pydantic import BaseModel, Field
 
-from .BaseModelFunctions import ytBaseModel, ytDataObjectAbstract, ytParameter
+from .base_model_functions import ytBaseModel, ytDataObjectAbstract, ytParameter
```

Comment:
Ok, I was wrong, I think we need a split, not a skip!

```suggestion
from .base_model_functions import ytBaseModel, ytDataObjectAbstract, ytParameter  # isort: split
```

Reply:
I'm just going to go ahead and commit it...